Assets/Scripts/FengGameManagerMKII.cs

```diff
@@ -159,6 +159,7 @@ public class FengGameManagerMKII : Photon.MonoBehaviour
     private bool startRacing;
     public int time = 600;
     private float timeElapse;
+    //should be better to be able to retrive this info somehow in order to be used in other place
     private float timeTotalServer;
```

Comment:
Rather mark this with an [Obsolete] attribute instead of a comment. Via obsolete your IDE will display a message when you are trying use the field, for a comment you'd have to look it up here.

Reply:
rodger